### PR TITLE
Read Me

### PR DESCRIPTION
**Pull request by kanitw**
cc: @mattwchun @AkshatSh

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,56 @@
 # CompassQL
-CompassQL Query Language for visualization recommendation.
+
+CompassQL is a visualization query language that powers chart recommendations in [Voyager 2](https://github.com/vega/voyager2).
+
+As described in our [vision paper](idl.cs.washington.edu/papers/compassql) and [Voyager 2](idl.cs.washington.edu/papers/voyager2) paper, a CompassQL query is a JSON object that contains the following components:
```
Comment (yhoonkim) [inline README.md]: Links that don't start with 'http(s)://' didn't work properly when I opened the file on Github.

File: README.md [hunk 2]
```diff
@@ -1,5 +1,56 @@
...
+
+```
+var schema = cql.schema.Schema.build(data);
+```
+
+(You can reuse the same schema for querying the same dataset multiple times.)
+
+2) Execute a CompassQL query.
+
+```
```
Comment (yhoonkim) [inline README.md]: I wish to have highlighted code by adding "js" after "```".

**Comment by kanitw** [comment]
Feel free to fix them as u suggested

**Comment by kanitw** [comment]
Thx

**Pull request merged**